### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by zach-bergman**
Route, landing page with form, welcome controller with index action. Search button directs to search index page. Tests for all!

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<h1>oompr</h1>
+
+<h3>Where do you want to live?</h3>
+
+<%= form_tag(search_path, method: :get) do %>
+  <%= text_field_tag :city, params[:city] %>
```
Comment (RodrigoACG) [inline app/views/welcome/index.html.erb:6]: should we have a place holder here ? Just so it is not an empty block.

File: spec/features/welcome/index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      expect(page).to have_button('Search')
+    end
+
+    it 'I click the search button, brings me to the search results page' do
+      visit root_path
+
+      fill_in :city, with: 'Houston'
+      click_button 'Search'
+
+      expect(current_path).to eq(search_path)
```
Comment (RodrigoACG) [inline spec/features/welcome/index_spec.rb:24]: testing looks good !!

File: app/views/welcome/index.html.erb (hunk 1 above)
Comment (jdmchugh111) [reply app/views/welcome/index.html.erb:6]: I think it makes sense for the search field to be empty - I haven't seen many websites that pre-populate a search field for you

**Review by jdmchugh111 — APPROVED** [review]
great work!

**Pull request merged**